src-ui/setup-jest.ts

```diff
@@ -76,7 +76,10 @@ const mock = () => {
   let storage: { [key: string]: string } = {}
   return {
     getItem: (key: string) => (key in storage ? storage[key] : null),
-    setItem: (key: string, value: string) => (storage[key] = value || ''),
+    setItem: (key: string, value: string) => {
+      if (value.length > 1000000) throw new Error('localStorage overflow')
```

Comment:
const maxLocalStorageSize = 1000000;
if (value.length > maxLocalStorageSize) throw new Error('localStorage overflow')

Reply:
Thanks but this is purely for testing so that change isn't very useful I think